website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -6,11 +6,61 @@ description: The `/sys/sync` endpoints are used to configure destinations and as
 
 # `/sys/sync`
 
+@include 'alerts/restricted-root.mdx'
```

Comment:
I think it's a bit too high and should be in the `## Configuration` section?

Reply:
Whoops, yes that's right